hibike/devices/Potentiometer/potentiometer.cpp

```diff
@@ -1,59 +1,51 @@
 #include "potentiometer.h"
-#include <Servo.h>
 
-
-
-uint8_t pins[NUM_PINS] = {IN_0, IN_1, IN_2, IN_3};
+uint8_t pins[NUM_PINS] = {IN_0, IN_1, IN_2};
```

Comment:
This implimentation of potentiometer has three devices.

The README.md specifies 4.  Please update README.md to match.